integration-tests/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/multiversx/firehose-multiversx/checker
 go 1.17
 
 require (
-	github.com/multiversx/firehose-multiversx/types v0.0.0-20230518130743-9686cd85912e
-	github.com/multiversx/mx-chain-core-go v1.2.4-0.20230517135533-2e54a17cd912
+	github.com/multiversx/firehose-multiversx/types v0.0.0-20240209135824-1372c8c9da18
```

Comment:
no actual release needed?

Reply:
some more changes will come, will update at the end with a proper release tag.